peps/pep-0772.rst

```diff
@@ -189,13 +188,14 @@ Election of the council
 
 A council election consists of two phases:
 
-* Phase 1: A member of the Packaging community (defined later in this document)
-  can self-nominate themselves for the council elections.
-* Phase 2: Each Packaging community member can assign zero to five stars to
-  each candidate. Voting is performed anonymously. The outcome of the vote is
-  determined using the `STAR voting system`_, modified to use the `Multi-winner
-  Bloc STAR`_ approach. If a tie occurs, it may be resolved by mutual agreement
-  among the candidates, or else the winner will be chosen at random.
+* Phase 1: A voting member (defined later in this document) can self-nominate
+  themselves for the council elections. Such a nomination must include
```

Comment:
```suggestion
  themselves, or nominate any other candidate, for the council elections. Such a nomination must include
```

Reply:
I agree and like the first suggestion, which I'll add in a follow up PR.  For SC elections, PEP 13 doesn't require a formal recorded acceptance, but the returns officer can validate such third party nominations if needed.